backend/application/favoriteVideo.ts

```diff
@@ -52,8 +52,33 @@ export class FavoriteVideoApplicationService {
       )
     }
   }
+  async getfavoriteVideos() {
+    try {
+      const favoriteVideos = await prisma.favoriteVideo.findMany({
+        where: {
+          deleted_at: null,
+        },
+        include: {
+          video: {
+            include: {
+              section: {
+                include: {
+                  course: {},
```

Comment:
```suggestion
                  course: true,
```

じゃないですか？
他のコードもそうなってる気がするんですがどうでしょう？